doc/stdlib/linalg.rst

```diff
@@ -1311,17 +1311,180 @@ Linear algebra functions in Julia are largely implemented by calling functions f
+
+.. function:: Ac_ldiv_B(A, B)
+
+   .. Docstring generated from Julia source
+
+   For matrices or vectors :math:`A` and :math:`B`\ , calculates :math:`Aᴴ` \\ :math:`B`\ .
+
+.. function:: Ac_ldiv_B!([Y,] A, B) -> Y
+
+   .. Docstring generated from Julia source
+
+   Similar to :func:`A_ldiv_B!`\ , but return :math:`Aᴴ` \\ :math:`B`\ , computing the result in-place in ``Y`` (or overwriting ``B`` if ``Y`` is not supplied).
+
+.. function:: Ac_ldiv_Bc(A, B)
+
+   .. Docstring generated from Julia source
+
+   For matrices or vectors :math:`A` and :math:`B`\ , calculates :math:`Aᴴ` \\ :math:`Bᴴ`\ .
+
+.. function:: Ac_mul_B(A, B)
+
+   .. Docstring generated from Julia source
+
+   For matrices or vectors :math:`A` and :math:`B`\ , calculates :math:`Aᴴ⋅B`\ .
+
+.. function:: Ac_mul_Bc(A, B)
+
+   .. Docstring generated from Julia source
+
+   For matrices or vectors :math:`A` and :math:`B`\ , calculates :math:`Aᴴ Bᴴ`\ .
+
+.. function:: Ac_rdiv_B(A, B)
+
+   .. Docstring generated from Julia source
+
+   For matrices or vectors :math:`A` and :math:`B`\ , calculates :math:`Aᴴ / B`\ .
+
+.. function:: Ac_rdiv_Bc(A, B)
+
+   .. Docstring generated from Julia source
+
+   For matrices or vectors :math:`A` and :math:`B`\ , calculates :math:`Aᴴ / Bᴴ`\ .
+
+.. function:: At_ldiv_B(A, B)
+
+   .. Docstring generated from Julia source
+
+   For matrices or vectors :math:`A` and :math:`B`\ , calculates :math:`Aᵀ` \\ :math:`B`\ .
+
+.. function:: At_ldiv_B!([Y,] A, B) -> Y
+
+   .. Docstring generated from Julia source
+
+   Similar to :func:`A_ldiv_B!`\ , but return :math:`Aᵀ` \\ :math:`B`\ , computing the result in-place in ``Y`` (or overwriting ``B`` if ``Y`` is not supplied).
+
+.. function:: At_ldiv_Bt(A, B)
+
+   .. Docstring generated from Julia source
+
+   For matrices or vectors :math:`A` and :math:`B`\ , calculates :math:`Aᵀ` \\ :math:`Bᵀ`\ .
+
+.. function:: At_mul_B(A, B)
+
+   .. Docstring generated from Julia source
+
+   For matrices or vectors :math:`A` and :math:`B`\ , calculates :math:`Aᵀ⋅B`\ .
+
+.. function:: At_mul_Bt(A, B)
+
+   .. Docstring generated from Julia source
+
+   For matrices or vectors :math:`A` and :math:`B`\ , calculates :math:`Aᵀ⋅Bᵀ`\ .
+
+.. function:: At_rdiv_B(A, B)
+
+   .. Docstring generated from Julia source
+
+   For matrices or vectors :math:`A` and :math:`B`\ , calculates :math:`Aᵀ / B`\ .
+
+.. function:: At_rdiv_Bt(A, B)
+
+   .. Docstring generated from Julia source
+
+   For matrices or vectors :math:`A` and :math:`B`\ , calculates :math:`Aᵀ / Bᵀ`\ .
+
 BLAS Functions
 --------------
 
 .. module:: Base.LinAlg.BLAS
 
-:mod:`Base.LinAlg.BLAS` provides wrappers for some of the BLAS functions for
-linear algebra.  Those BLAS functions that overwrite one of the input
-arrays have names ending in ``'!'``.
-
-Usually a function has 4 methods defined, one each for ``Float64``,
-``Float32``, ``Complex128`` and ``Complex64`` arrays.
+In Julia and most other scientific computing, linear-algebra operations are
```

Comment:
I changed it to "In Julia (as in much of scientific computation), dense linear-algebra ..."